wiv.js

```diff
@@ -22,45 +22,82 @@ function wiv(params) {
 
   function initWiv(wiv) {
     //style wiv elements
+    wiv.meta = {};
     wiv.style.display = "inline-block";
-    wiv.style.borderRadius = parseFloat(wiv.dataset.wivHeight) + "px";
-    wiv.children[0].style.padding = (parseFloat(wiv.dataset.wivHeight) * 4) + "px";
+    let wivContent = document.createElement('div');
```

Comment:
This is to enforce one-child-element-per-wiv, which has always been necessary for this library to function properly. Now, users can throw whatever they want to in a wiv, and the library will bundle it into a single content wiv for them.

Reply:
🔥  bonus